### ISSUE TITLE
Applying order weight during Meta post import

### ISSUE THREAD

**Issue by erlend-sh** [erlend-sh/learndiscourse#20]
> `weight` in every doc meta info is used to order things. So you can put a weight value on them. More weight will show higher.

So with the [concept of weight](https://github.com/nahtnam/learndiscourse/issues/1#issuecomment-127587872), could we perhaps apply this automatically to articles, based on the order by which they are imported? This would make it a whole lot easier to reorder article, since we'd just have to cut & paste links around in the `doc_list.yml`, as opposed to going into individual articles to tweak weight values.

**Comment by erickguan** [comment]
That's odd since you don't often order every docs. But it's possible to save the weight or whatever meta info from the original file. So the meta attrs won't lost after re-importing.

**Comment by erlend-sh** [comment]
I'm not sure if we completely understand each other here. This is probably a more suitable topic for Gitter/PM chat, since I suspect there might be some technical aspects of this that I don't grasp well enough yet to explain myself as well as I'd like to.

> But it's possible to save the weight or whatever meta info from the original file. So the meta attrs won't lost after re-importing.

Not quite following. Are you suggesting that we include some meta-info on the original Meta posts? I think this could be useful in some instances, but in this case I think we'll have the same problem with tedious reordering.

**Comment by erickguan** [comment]
I means you can add the weight attr to the file. Then the script can read that weight and save back to the downloading file when importing as if the filename matches. Is that make sense to you?

**Comment by erlend-sh** [comment]
It's starting to make sense to me, but _where_ do you add this weight attribute for the script to pick it up?

**Comment by erickguan** [comment]
1. The script downloads doc from Meta.
2. The script calculate doc name.
3. Read the original file in `_en` if exists.
4. Copy the weight attr to the new downloaded file.

So it depends on the original filename on the same subsection!

**Comment by erlend-sh** [comment]
Ooooh. So it will actually look at one of the _originals_! Alright, it's coming together for me now :grinning: 

I did however just put that to the test, and couldn't get it to work.
1. I assigned a bunch of random `weight: n` attributes to all files in the original /sysadmin/essentials folder.  e.g:

```
---
title: Set up reply via email support
weight: 60
---
```
1. I ran the import script
2. None of the generated files had adopted the new weight attribute.
